### ISSUE TITLE
Option to change order of reminders list

### ISSUE THREAD

**Issue by 123d57** [DamascenoRafael/reminders-menubar#21]
Hey, 
Is there a way you could add the option of changing the order of the reminder sub-categories? Rather than (from what I assume) being in alphabetic order. 

Love the app
Cheers

**Comment by DamascenoRafael** [comment]
@123d57 Sounds like a good idea, in fact the lists are sorted alphabetically.
I just don't know where we could put this option. Any idea how we can fit this into the interface?

**Comment by hoobtron** [comment]
by default it should adopt the set sort order of lists from the Reminders app

Additionally nice to have:
The ability to change sorting of lists in the check-lists dropdown (left bottom button) by moving the title of one sub-category to another position

**Comment by dartiss** [comment]
For me, this is a bigger issue because by sorting by alphabetical order it's showing reminders differently than in my Reminders app.

Reminders lets you drag and drop items into any order, whereas this app just blanket shows them alphabetically, which is not what I want.

**Comment by DamascenoRafael** [comment]
In version 1.9.0 📦 reminder lists are sorted like in Apple Reminders.
Can you try and verify the behavior?

I tried to implement an option to change the order manually, but I couldn't do it directly through the filter dropdown (left bottom button).

**Comment by jeremybagel** [comment]
Drag and drop would be amazing for me - although this is certainly working where it mirrors the Reminders app itself, it would be good if the function worked in duplex so that we could drag and drop them directly within the menu bar and it would update/reflect back in the main app.

Brilliant work - and thanks for creating this for us.

**Comment by DamascenoRafael** [comment]
Hey, I spent some time thinking about this, but I think as it stands today we can close this issue.

Currently (thanks to the suggestions in this issue) the lists appear in the same order as in the Apple Reminders app.

I considered allowing changing the order of the lists through the app, but that way if a new list was created or the order was changed through the Apple Reminders app we would have to decide how to handle it and in the end the two apps would always end up out of sync .

The best solution would be, as suggested, to run in duplex reflecting the changes between the 2 apps. However, the Apple EventKit API is very limited and does not allow defining such a thing.
